library/src/main/java/com/schibsted/spain/barista/internal/matcher/TextMatchers.kt

```diff
@@ -0,0 +1,12 @@
+package com.schibsted.spain.barista.internal.matcher
+
+import android.support.test.espresso.matcher.ViewMatchers.withText
+import android.view.View
+import org.hamcrest.CoreMatchers.containsString
+import org.hamcrest.Matcher
+
+object TextMatchers {
+
+    @JvmStatic
+    fun withContainsText(text: String): Matcher<View> = withText(containsString(text))
```

Comment:
This class and method don't give much value. We should avoid adding internal aliases, it just adds noise for future readers.
Let's remove it.

Reply:
Done